ui/apps/platform/src/Containers/ComplianceEnhanced/ScanConfigs/compliance.scanConfigs.utils.ts

```diff
@@ -0,0 +1,120 @@
+import {
+    DailySchedule,
+    MonthlySchedule,
+    Schedule,
+    ScheduleBase,
+    UnsetSchedule,
+    WeeklySchedule,
+} from 'services/ComplianceEnhancedService';
+import { getDayOfMonthWithOrdinal, getTimeHoursMinutes } from 'utils/dateUtils';
+
+import { ScanConfigFormValues, ScanConfigParameters } from './Wizard/useFormikScanConfig';
+
+export function convertFormikParametersToSchedule(parameters: ScanConfigParameters): Schedule {
+    const { intervalType, time, daysOfWeek, daysOfMonth } = parameters;
+
+    // Convert the time to hour and minute
+    const [hourString, minuteString] = time.split(/[: ]+/);
+    let hour = parseInt(hourString);
+    const minute = parseInt(minuteString);
```

Comment:
Small style suggestion to include explicit `10` for decimal radix as second argument.

Reply:
Updated